cmd/server/temporal/server.go

```diff
@@ -190,11 +190,14 @@ func (s *server) startService() common.Daemon {
 		}
 
 		params.PublicClient, err = sdkclient.NewClient(sdkclient.Options{
-			HostPort:          s.cfg.PublicClient.HostPort,
-			Namespace:         common.SystemLocalNamespace,
-			MetricsScope:      params.MetricScope,
-			Logger:            l.NewZapAdapter(zapLogger),
-			ConnectionOptions: sdkclient.ConnectionOptions{TLS: options},
+			HostPort:     s.cfg.PublicClient.HostPort,
+			Namespace:    common.SystemLocalNamespace,
+			MetricsScope: params.MetricScope,
+			Logger:       l.NewZapAdapter(zapLogger),
+			ConnectionOptions: sdkclient.ConnectionOptions{
+				TLS:                options,
+				DisableHealthCheck: true,
```

Comment:
Why were we disabling health checks before?

Reply:
It is new option I recently added to Go SDK. And we are creating client before starting server, so it fais right away.